File: StykerDG.FarmForge.DataModel/Models/Status.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace StrykerDG.FarmForge.DataModel.Models
{
    [Table("Status")]
    public class Status : BaseModel
    {
        [Key]
        public int StatusId { get; set; }
        public string EntityType { get; set; }
        public string Name { get; set; }
        public string Label { get; set; }
    }
}

File: StrykerDG.FarmForge.Client/farmforge_client/lib/models/general/status.dart
class Status {
  int statusId;
  String entityType;
  String name;
  String label;

  Status({
    this.statusId,
    this.entityType,
    this.name,
    this.label
  });

  factory Status.fromMap(Map<String, dynamic> data) {
    return Status(
      statusId: data['StatusId'],
      entityType: data['EntityType'],
      name: data['Name'],
      label: data['Label']
    );
  }

  Map<String, dynamic> toMap() {
    return {
      'StatusId': this.statusId,
      'EntityType': this.entityType,
      'Name': this.name,
      'Label': this.label
    };
  }
}